src/language.rs

```diff
@@ -122,6 +122,22 @@ pub enum Language {
 }
 
 impl Language {
+    /// Construct a word list from its language code. Returns None
+    /// if the language code is not valid or not supported.
+    pub fn from_language_code(language_code: &str) -> Option<Self> {
+        match &language_code.to_ascii_lowercase()[..] {
+            "en" => Some(Language::English),
+            "zh-hans" => Some(Language::ChineseSimplified),
+            "zh-hant" => Some(Language::ChineseTraditional),
+            "fr" => Some(Language::French),
+            "it" => Some(Language::Italian),
+            "ja" => Some(Language::Japanese),
+            "ko" => Some(Language::Korean),
+            "es" => Some(Language::Spanish),
```

Comment:
All of these except `English` will fail to compile with default features disabled. We should probably add tests without default features to CI.